docs/Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md

```diff
@@ -81,15 +81,29 @@ Create the following environment variables and set to the specified values:
   Besu uses the specified node to deploy the contracts and is the first node in the network.
 * `CHAIN_ID` The chainID from the genesis file.
 
+!!! tip
+
+    A simple way to set multiple environment variables is to create a file called `.env` with the required settings
+
+```env
+NODE_INGRESS_CONTRACT_ADDRESS=0x0000000000000000000000000000000000009999
+ACCOUNT_INGRESS_CONTRACT_ADDRESS=0x0000000000000000000000000000000000008888
+BESU_NODE_PERM_ACCOUNT=627306090abaB3A6e1400e9345bC60c78a8BEf57
+BESU_NODE_PERM_KEY=c87509a1c067bbde78beb793e6fa76530b6382a4c0241e5e4a9ec0a0f44dc0d3
+CHAIN_ID=2018
+```
```

Comment:
Might be best to shift this block 4 spaces across to include it in the tip.

Reply:
done